src/Server.php

```diff
@@ -152,7 +150,7 @@ public function setClass($class, $namespace = '', $argv = null)
         $reflection = Reflection::reflectClass($class, $argv, $namespace);
 
         foreach ($reflection->getMethods() as $method) {
-            $definition = $this->_buildSignature($method, $class);
+            $definition = $this->buildSignature($method, $class);
```

Comment:
Is this due to the changed parent contract?

Reply:
Underscore method was deprecated and proxies to renamed copy in parent. This could be a BC break if this class was extended and underscored method was overridden. I'll revert.